.github/workflows/ubuntu-2004.yml

```diff
@@ -154,6 +154,13 @@ jobs:
           export DOCKER="docker run --rm -v ${GITHUB_WORKSPACE}:${GITHUB_WORKSPACE} -w ${GITHUB_WORKSPACE} ${UBUNTU_2004_IMAGE}"
           docker pull ${UBUNTU_2004_IMAGE}
           ${DOCKER} bash -c 'cd build && ctest --output-on-failure -j $(nproc) -LE "nonparallelizable_tests|long_running_tests|wasm_spec_tests"'
+      - uses: actions/upload-artifact@v3
+        with:
+          name: parallel-logs
+          path: |
+            build/var/*
```

Comment:
I went to the artifacts page https://github.com/eosnetworkfoundation/mandel/actions/runs/2580037524#artifacts for this PR's CICD and downloaded `forked-chain-logs` and `irreversible-logs`. I only saw `var/subprocess_results.log` and `etc/eosio/launcher/testnet.template`. I did not see directories like `var/lib/node_00`, `etc/eosio/node_00`, ...

Not sure you need to specify all the directory levels like `build/var/lib/*/*` to get `build/var/lib/node_00`, or just `build` to get the whole directory. Or those node directories were somehow deleted.

Reply:
They are cleaned up by the tests on successful runs. We might choose to add `--keep-logs` to our tests in the future so they are preserved for successful runs, but we do not current do that.

Reply:
Just kicked off: https://github.com/eosnetworkfoundation/mandel/pull/567 so we can verify logs are saved on failed tests.

Reply:
I did not see logs for the failed tests in https://github.com/eosnetworkfoundation/mandel/actions/runs/2586917690#artifacts.